pkgs/tools/security/passage/default.nix

```
@@ -22,6 +24,13 @@ stdenv.mkDerivation {
     sha256 = "1val8wl9kzlxj4i1rrh2iiyf97w9akffvr0idvbkdb09hfzz4lz8";
   };
 
+  patches = lib.optional stdenv.isDarwin [
```

```suggestion
  patches = [
```

Or, in the worst case, `lib.optionals stdenv.isDarwin [`.